library/src/main/java/org/apache/apex/malhar/lib/state/managed/ManagedTimeUnifiedStateImpl.java

```diff
@@ -59,22 +60,22 @@ public ManagedTimeUnifiedStateImpl()
   }
 
   @Override
-  public int getNumBuckets()
+  public long getNumBuckets()
   {
     return timeBucketAssigner.getNumBuckets();
   }
 
   @Override
   public void put(long time, @NotNull Slice key, @NotNull Slice value)
```

Comment:
Can you please call out in the class level javadoc how this class differs from ManagedTimeStateImpl?